lxd/instance/drivers/load.go

```diff
@@ -81,6 +81,33 @@ func validDevices(state *state.State, cluster *db.Cluster, instanceType instance
 		if err != nil {
 			return errors.Wrap(err, "Failed detecting root disk device")
 		}
+
+		for deviceName, device := range devices {
```

Comment:
@freeekanayaka I wonder is this possible to push down into the relevant device's `validateConfig` function? As we've been trying to avoid putting device specific code in the generic loaders.

Reply:
Where would that be? Is there a single spot that we can plug this check into or would we risk to have to duplicate in various devices? Apologies, as I'm not very familiar with this code.